bedrock/careers/templates/careers/benefits.html

```diff
@@ -7,124 +7,156 @@
 {% extends "careers/base.html" %}
 
 {% block careers_content %}
-
-<!-- Hero -->
   <header class="c-careers-text-hero">
     <div class="mzp-l-content c-text-hero-wrapper mzp-t-content-md">
-      <div class="c-text-hero-title">
-        <h1>Benefits for real life</h1>
-      </div>
+      <h1 class="c-text-hero-title">Benefits for real life</h1>
+
       <div class="c-text-hero-body">
-        <p>We’re proud to offer our employees these outstanding benefits, which set us apart from other companies in our industry.</p>
+        <p>We’re proud to offer our employees these outstanding benefits, which
+          set us apart from other companies in our industry.</p>
 
-        <div class="c-careers-button-wrapper">
+        <p class="c-careers-button-wrapper">
           <a href="{{ url('careers.listings') }}" class="mzp-c-button mzp-t-secondary" data-cta-type="button" data-cta-name="careers listings">See our open roles</a>
-        </div>
+        </p>
       </div>
     </div>
   </header>
 
   <section class="c-careers-benefits-intro c-careers-section">
-    <div class="mzp-l-content mzp-t-content-md">
-      <h2>What we offer</h2>
+    <div class="mzp-l-content mzp-t-content-xl">
+      <div class="c-careers-section-intro">
+        <h2 class="c-careers-section-title">
```

Comment:
Previously all h2s were centered but it's more reliable to define a KIND of heading and use a class. For example there might be some h2s that shouldn't be centered, or maybe the thing you want centered isn't an h2.